src/feature/gui/gui-runner.c

```diff
@@ -540,7 +558,8 @@ void mGUIRun(struct mGUIRunner* runner, const char* path) {
 					if (showOSD) {
 						unsigned origin = runner->params.width - GUIFontHeight(runner->params.font) / 2;
 						unsigned w;
-						if (fastForward || (heldKeys & (1 << mGUI_INPUT_FAST_FORWARD_HELD))) {
+						bool fastForwarding = fastForward || (heldKeys & (1 << mGUI_INPUT_FAST_FORWARD_HELD));
```

Comment:
This is fine to merge as-is, but I'm curious if there was any point to this change or if it just got confused as it got partially backed out?

Reply:
I believed this change was made because the variable was used in more places in earlier revisions of this pull request. As of now, it doesn't really add anything. Perhaps it's a bit easier to read like this, but that's a personal preference.
If you prefer this changed as well, let me know.

Reply:
It is possibly easier to read but it's also changing code without a functional difference in a PR that has functional changes, which I prefer to avoid. It makes reading git blame more complicated, for example. So if you could revert this one specific part I'll go ahead and merge it.

Reply:
Alright, I reverted this specific part!